tests/test.rs

```diff
@@ -285,7 +297,9 @@ fn test_integration() {
             }
 
             println!("\nLLVM");
-            let llvm_version = get_version("llvm-config", "--version");
+            let llvm_version = get_version("llvm-config");
+            let clang_version = get_version("clang++");
+            assert_eq!(llvm_version, clang_version, "llvm-config ({:?}) and clang++ ({:?}) have not the same major version", llvm_version, clang_version);
```

Comment:
Nit: if you make `get_compiler_major` return a String, you can replace `{:?}` with `{}`.

Also `have not the same major version` -> `don't have the same major version`.